rest/rest_actions.py

```diff
@@ -34,4 +33,17 @@ def create_issue_rest(self, path, issue_data):
             error = response.json()['errors']['summary']
         return response.status_code, key_issue, id_issue, error
 
+    def update_issue_rest(self, path, issue_data):
+        response = requests.put(self.HOST + path, auth=self.AUTH, data=issue_data, headers=self.HEADERS)
+        return response.status_code
+
+    def get_issue_rest(self, path):
+        response = requests.get(self.HOST + path, auth=self.AUTH, headers=self.HEADERS)
+        return response.status_code, response.json()
```

Comment:
Better always return Response object, this will improve readability and will create a skeleton of your code.